deploy/infrastructure/dependencies/terraform-aws-kubernetes/AWSLoadBalancerControllerPolicy.json

```diff
@@ -0,0 +1,219 @@
+{
+  "Version": "2012-10-17",
```

Comment:
I'm not sure I know what this file is, but this date seems very old -- is it still the appropriate value?

Reply:
This is an AWS IAM policy and the version is actually the latest according to the documentation: https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html

This file is actually a recommended policy provided by AWS. There is a comment in the code referencing the source of it: https://github.com/interuss/dss/pull/942/files/6d44621de77fd97abd4d9425db1c69ece979b4cf#diff-18a9f1d810cf980f53606b3f711101b74f9335fae88cfcaa0b921e46049d8218R53-R54